src/gui/AutomationPatternView.cpp

```diff
@@ -270,17 +269,22 @@ void AutomationPatternView::paintEvent( QPaintEvent * )
 	if( !( m_pat->getTrack()->isMuted() || m_pat->isMuted() ) )
 		c = styleColor;
 	else
-		c = QColor( 80, 80, 80 );
+		c = mutedColor();
 
 	if( isSelected() == true )
 	{
-		c.setRgb( qMax( c.red() - 128, 0 ), qMax( c.green() - 128, 0 ), 255 );
+		c = selectedColor();
 	}
 
 	lingrad.setColorAt( 1, c.darker( 300 ) );
 	lingrad.setColorAt( 0, c );
-
-	p.setBrush( lingrad );
+	
+	if ( gradient() ) {
+		p.setBrush( lingrad );
+	} else {
+		p.setBrush( c );
+	}
+	
```

Comment:
How about putting the whole if statement into one line?

`if( gradient() ) { p.setBrush( lingrad ); } else { p.setBrush( c ); }`